libcxx/test/benchmarks/containers/ContainerBenchmarks.h

```diff
@@ -39,6 +39,36 @@ void BM_CopyConstruct(benchmark::State& st, Container) {
   }
 }
 
+template <class Container>
+void BM_MoveConstruct(benchmark::State& st, Container) {
+  auto size = st.range(0);
+  Container c(size);
+  for (auto _ : st) {
+    auto v = std::move(c);
```

Comment:
This benchmark has the issue that it only moves from a non-empty vector the first time around. After the first iteration, it is moving from a moved-from vector.

There's also the cost of destruction inside this benchmark, which may or may not add too much noise.

Since your patch doesn't actually change the move constructor (and the move constructor is close to trivial), I don't think you need this benchmark as part of this patch at all. I would suggest removing it.